frontend/public/data/possibleFilters.js

```diff
@@ -3,23 +3,34 @@ import ExploreOutlinedIcon from "@material-ui/icons/ExploreOutlined";
 import GroupIcon from "@material-ui/icons/Group";
 import GroupAddIcon from "@material-ui/icons/GroupAdd";
 import LocationOnOutlinedIcon from "@material-ui/icons/LocationOnOutlined";
-import { useContext } from "react";
-import UserContext from "../../src/components/context/UserContext";
 import getTexts from "../texts/texts";
 
-export default function getFilters(key, filterChoices) {
-  const { locale } = useContext(UserContext);
+export default function getFilters({ key, filterChoices, locale }) {
   const texts = getTexts({ page: "filter_and_search", locale: locale });
+  const english_texts = getTexts({ page: "filter_and_search", locale: "en" });
   if (!filterChoices) {
     throw new Error("No filter choices supplied");
   }
 
   if (key === "projects") {
-    return getProjectsFilters(filterChoices, texts);
+    return getProjectsFilters(filterChoices, texts, english_texts);
   } else if (key === "organizations") {
     return getOrganizationsFilters(filterChoices, texts);
   } else if (key === "members") {
     return getMembersFilters(filterChoices, texts);
+  } else if (key === "ideas") {
+    return getIdeasFilters(filterChoices, texts);
+  } else if (key === "all") {
+    const projectsFilters = getProjectsFilters(filterChoices, texts, english_texts);
+    const organizationsFilters = getOrganizationsFilters(filterChoices, texts);
+    const membersFilters = getMembersFilters(filterChoices, texts);
+    return [
+      ...projectsFilters,
+      ...organizationsFilters.filter((o) => !projectsFilters.map((p) => p.key).includes(o.key)),
```

Comment:
Why do we need an additional `.filter` and `.map` here? Do you mind adding a one-line comment for future us?